grass7/raster/r.richdem.flowaccumulation/r.richdem.flowaccumulation.py

```diff
@@ -130,4 +129,5 @@ def main():
     accum.write(_output, overwrite=gscript.overwrite())
 
 if __name__ == "__main__":
+    options, flags = gscript.parser()
```

Comment:
Why this change? Plus it seems unrelated to the lazy import.

Reply:
The module depends on `richdem` library. The compilation (`--html-description`) fails when if `parser()` is called in `main()` function because of missing library on build server (linux logs, windows binaries).